sig-apps/Charter.md

```diff
@@ -0,0 +1,141 @@
+# SIG Apps Charter
+
+## Scope
+
+SIG Apps Covers deploying and operating applications in Kubernetes with a focus on the application developer and application operator experience.
+
+This includes but is not limited to:
+
+* Discussions on all levels from application development, testing environments, and production
+* Discussion of how to define and run apps in Kubernetes
+* Demos of relevant tools and projects
+* Discussion on areas of friction that can lead to suggesting improvements or feature requests
+* Development of Kubernetes primitives (e.g., Workloads API) to enable the application developers and operators. Note, most new types should be created as CRDs during their development phase and may always be ecosystem projects
```

Comment:
e.g. should be i.e.

Reply:
@r2d4 In this case it's for example (e.g.) rather than that is (i.e.). Jobs is something else SIG Apps owns. There's also the application crd/controller which is being developed under SIG Apps for interoperability purposes.

Reply:
We also own PDB in the policy group. We may own other portions of the API, where appropriate, in the future. We do expect that most development of primitives will now occur as extensions, but as with DeployConfig -> Deployment, when there is universal consensus to move a primitive into a core API type, we may do so.